src/java.base/share/classes/jdk/internal/perf/Perf.java

```diff
@@ -298,7 +242,7 @@ public void run() {
      * @throws  OutOfMemoryError The instrumentation buffer could not be mapped
      *                           into the virtual machine's address space.
      */
-    private native ByteBuffer attach0(String user, int lvmid)
+    private native ByteBuffer attach0(int lvmid)
                    throws IllegalArgumentException, IOException;
```

Comment:
You can drop the "throws IllegalArgumentException" here if you want, it's not needed as it's a runtime exception.

Reply:
Fixed.